chef/api.py

```diff
@@ -228,13 +213,13 @@ def api_request(self, method, path, headers={}, data=None):
             headers['content-type'] = 'application/json'
             data = json.dumps(data)
         response = self.request(method, path, headers, data)
-        return json.loads(response.decode())
+        return response.json()
 
     def __getitem__(self, path):
         return self.api_request('GET', path)
 
 
-def autoconfigure(base_path=None):
+def autoconfigure(base_path=None, verify_ssl=True):
```

Comment:
This shouldn't be passed in here, it should parse the value from `knife.rb`. like everything else.